docs/assets/custom.css

```diff
@@ -117,6 +117,10 @@ aside > div > ul {
     border: solid 1px #ddd;
     border-radius: 0 6px 6px 6px;
     box-shadow: 0 0 10px rgba(0, 0, 0, .05);
-    padding: 4em 2em;
+    padding: 2em;
   }
 }
+
+.dark-mode .dark\:prose-dark .tabs-component-panels  {
+  color: #4a5568;
+}
```

Comment:
Could we make the whole tab panel dark?

Reply:
Not the entire tab panel to dark, only paragraph inside of it with `prose-dark` selector 😀
It's currently illisible because of very very light gray

Reply:
Yes, I understand. But I think it would look even better if the whole tab were dark....